src/main/java/com/fynd/extension/model/Extension.java

```diff
@@ -98,6 +102,17 @@ public PlatformClient getPlatformClient(String companyId, AccessToken session) {
         PlatformConfig platformConfig = this.getPlatformConfig(companyId);
         platformConfig.getPlatformOauthClient()
                       .setToken(session);
+        if(session.getExpiresIn()!=0) {
+            if (((session.getExpiresIn() - new Date().getTime()) / 1000) <= 120) {
```

Comment:
@aparnab93 Please read 120 from some config property value

Reply:
The value is never going to change. Can I make a static final variable and use it?